app/database/connection.py

```diff
@@ -0,0 +1,13 @@
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+
+class Connection():
+    _Session = None
+
+    @classmethod
+    def get(cls, url):
+        if cls._Session is None:
+            engine = create_engine(url, echo=False)
+            cls._Session = sessionmaker(bind=engine)
+        return cls._Session()
```

Comment:
You might consider a separate initialization method that takes the url parameter, and that can only be called once. Something like
```python
@classmethod
def init(cls, url):
    if cls._Session is None:
        engine = create_engine(url, echo=False)
        cls._Session = sessionmaker(bind=engine)
    else:
        raise Exception('Init can only be called once!')

@classmethod
def get(cls):
    return cls._Session()
```
I like that pattern better because it makes it clearer that the `url` param is only used once, and would also mean you wouldn't have to pass the db url string all around the app.